crates/asap/src/lib.rs

```diff
@@ -304,34 +317,227 @@ mod tests {
1,4.3,5.2,7.9,10.8,14.4,14.6,15.3,13.4,9.1,2.8,5.3,7.5,3.8,3.3,8.2,11.6,11.8,15.3,16.4,13.0,10.6,6.8,1.9,1.6,0.9,3.2,5.4,12.8,15.2,16.1,15.3,14.0,7.5,7.8,2.3,3.8,6.5,5.7,6.7,13.0,13.3,15.4,16.1,11.9,9.3,5.5,6.9,2.9,1.9,3.6,7.1,13.5,14.3,13.9,15.7,12.7,7.4,3.3,5.5,5.2,6.0,7.2,8.2,11.8,14.4,14.1,13.6,13.0,10.4,6.8,4.2,7.3,4.8,7.4,8.0,11.5,14.7,18.5,15.4,14.1,12.8,4.6,6.5,3.7,4.4,4.6,5.5,12.7,13.8,13.7,13.6,12.2,8.2,5.9,5.8,5.6,5.6,6.5,7.6,9.2,12.5,17.5,15.2,12.5,9.7,3.3,3.8,4.7,3.3,4.1,6.9,11.6,13.9,15.3,14.1,13.3,10.1,7.1,6.8,5.3,5.2,4.9,7.5,11.6,15.0,16.8,15.4,11.5,10.4,3.6,2.8,4.6,6.8,6.3,9.3,10.2,13.6,17.1,16.2,14.4,8.1,5.9,4.2,4.6,3.9,7.3,7.9,11.2,12.6,15.9,15.7,12.5,10.5,6.2,2.1,5.2,5.8,6.3,8.5,10.9,12.9,16.1,15.3,12.8,10.1,7.6,3.4,1.3,0.4,6.2,6.8,11.3,13.3,16.0,15.4,15.3,9.6,6.7,5.8,5.6,2.5,5.3,8.3,10.7,15.3,15.2,15.7,13.6,10.5,6.2,4.3,3.2,3.9,4.2,7.7,11.4,14.4,15.3,14.4,11.5,8.8,7.8,5.3,6.3,2.9,4.7,6.9,10.5,14.1,16.1,17.1,12.9,8.8,6.6,5.7,2.2,4.3,7.3,8.8,12.2,15.6,17.8,17.6,14.9,10.1,5.6,1.6,4.1,3.8,4.8,8.0,11.3,14.9,18.2,15.4,14.6,10.6,6.1,8.1,4.5,5.8,6.6,8.2,9.9,15.1,17.1,16.6,13.6,9.5,6.9,2.8,3.7,2.6,7.1,6.3,11.5,14.7,15.3,16.1,14.4,9.4,5.5,5.3,5.2,5.6,3.6,9.2,12.2,14.1,16.1,16.9,13.4,10.4,5.1,3.0,5.7,5.1,9.1,7.6,10.7,14.4,15.2,16.3,13.8,10.5,9.4,4.4,4.2,5.6,5.8,8.8,11.4,14.2,15.5,16.4,14.2,8.2,8.7,3.2,1.4,2.6,6.0,8.7,12.5,16.4,15.1,15.6,12.8,9.6,6.9,3.8,0.5,3.5,5.1,6.4,9.4,15.1,17.3,14.7,14.5,10.4,6.6,5.6,0.9,0.1,5.2,9.2,11.1,14.4,15.5,16.6,13.6,10.4,4.9,6.7,4.9,6.1,6.5,10.5,11.8,14.4,16.4,16.1,13.3,10.6,6.3,3.5,5.8,3.6,5.2,10.2,11.4,13.5,16.5,17.0,12.5,9.3,6.2,3.6,0.4,7.1,7.9,10.1,12.2,14.6,16.7,15.9,14.4,11.9,7.2,4.9,2.7,5.9,5.1,9.9,10.7,13.1,16.3,14.7,14.0,9.8,8.1,3.1,2.2,1.9,3.6,8.6,13.5,15.5,17.0,18.6,14.9,10.6,7.2,5.1,5.4,4.7,8.3,9.0,11.4,13.5,15.8,15.1,13.8,10.1,7.3,5.7,5.5,5.7,5.1,10.0,11.2,15.3,17.4,16.8,16.3,11.7,6.6,5.8,4.2,5.3,7.4,7.6,11.3,16.2,15.9,15.6,12.9,9.6,5.7,1.2,3.9,3.7,4.1,6.8,10.1,14.0,16.3,14.8,14.1,9.4,8.5,5.5,2.7,3.4,6.6,9.6,13.4,14.4,16.8,15.8,10.7,8.8,4.1,2.8,3.3,4.3,5.6,7.3,12.6,14.4,15.5,16.2,13.8,9.7,8.5,6.9,2.9,2.6,5.8,7.6,11.2,13.4,14.2,14.6,12.7,11.9,6.9,6.8,2.6,1.2,3.2,9.3,9.7,13.8,17.7,18.1,14.2,9.2,7.0,5.4,3.6,0.2,6.2,6.9,11.7,13.1,15.8,13.5,14.3,9.4,6.0,5.7,5.5,5.3,9.2,8.9,10.3,15.2,16.3,15.4,12.5,10.8,6.4,4.5,3.4,4.7,3.7,7.4,11.1,14.1,15.9,15.8,15.1,10.8,6.4,4.7,1.6,4.4,7.3,9.4,12.8,15.2,17.3,17.2,14.9,12.6,7.1,6.0,3.8,4.1,6.4,8.9,12.8,16.1,15.1,15.0,13.1,10.3,7.3,3.9,3.9,6.9,8.2,10.0,11.0,14.4,15.2,15.4,15.2,10.9,6.0,2.2,4.3,4.4,2.8,7.7,10.3,13.7,15.1,14.5,12.6,10.4,5.5,1.8,2.1,0.7,6.0,8.7,10.6,14.9,15.2,14.3,12.9,11.1,8.2,2.6,3.4,4.5,4.3,8.7,13.3,13.8,16.1,15.5,14.1,8.9,7.4,3.6,3.3,3.1,5.2,8.0,11.7,14.7,14.0,14.9,12.3,11.0,4.5,4.7,2.9,5.7,6.5,7.2,11.0,15.4,15.0,14.7,13.8,10.1,5.6,5.5,4.5,5.4,7.0,7.7,10.4,14.0,16.7,15.7,13.5,10.8,5.4,4.2,4.4,1.9,6.3,8.1,9.8,14.8,15.0,15.4,13.9,12.5,6.5,3.0,5.5,1.0,3.3,7.4,11.2,13.9,16.8,16.4,13.9,13.0,5.4,3.3,3.7,2.9,3.7,6.7,13.0,16.4,15.2,16.0,14.4,10.7,7.8,4.3];
```

Comment:
I think this might be the only one I don't like.. I don't think anyone is actually reading this data and a 200 line wall of numbers doesn't seem obviously better than a hilariously long single line.

Reply:
Undid this change and added a `rustfmt::skip`